### PR TITLE
Ui feature

### PR DESCRIPTION
**Pull request by Urchmaney**
Update the User interface.

- Add header component.
- Add styling for each components.

### CODE REVIEW

File: src/components/book-form.js
Comment (SunnyAsar) [inline src/components/book-form.js:19]: This is a whole lot of css, consider looking at `styled components` if you want you styles in here or just get external file for larger styles.

File: src/components/app.js
Comment (SunnyAsar) [inline src/components/app.js:7]: When the book list is not filled, the whole thing shifts down, leaving some space at the top.

Screenshot 2019-12-04 at 10 01 32 AM

**Review by SunnyAsar — APPROVED** [review]
# Hello @Urchmaney  and @fatymahmed  👋 

### Fantastic work guys 👏 
- Below i have pointed out an update we should make and the thing about css is a suggestion.
- Also we can make our repo `readme` better. As it is, there are no instructions on how to setup the repo, let's updated that.

No need for another review, good job on the PR description but the repo should be updated before merge.

**After the repo update**

## You May Pass 👍

**Comment by fatymahmed** [comment]
> # Hello @Urchmaney and @fatymahmed wave
> ### Fantastic work guys clap
> 
>     * Below i have pointed out an update we should make and the thing about css is a suggestion.
> 
>     * Also we can make our repo `readme` better, as it is there are not instructions on how to setup the repo, let's updated that.
> 
> 
> No need for another review, good job on the PR description but the repo should be updated before merge.
> 
> **After the repo update**
> ## You May Pass +1

Hi @SunnyAsar, thanks, we've made the changes suggested above.

**Pull request merged**
